samples/bot-people-picker-adaptive-card/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,53 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.11",
+  "version": "1.0.0",
+  "id": "{{Microsoft-App-Id}}",
+  "packageName": "com.microsoft.teams.typeaheadsearchcard",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "People picker card",
+    "full": "Adaptive card with people picker."
+  },
+  "description": {
+    "short": "Adaptive card with people picker",
+    "full": "This sample shows the feature of people picker on Adaptive Cards."
+  },
+  "icons": {
+    "outline": "outline.png",
+    "color": "color.png"
+  },
+  "accentColor": "#60A18E",
+  "bots": [
+    {
+      "botId": "{{Microsoft-App-Id}}",
+	    "needsChannelSelector": false,
+      "scopes": [
+        "personal","team","groupchat"
+      ],
+      "commandLists": [
+        {
+          "scopes": [
+            "personal"
+          ],
+          "commands": [
+            {
+              "title": "peoplepickercard",
+              "description": "To get people picker card"
+            }
+          ]
+        }
+      ],
+      "isNotificationOnly": false
+    }
+  ],
+  "permissions": [ "identity", "messageTeamMembers" ],
+  "validDomains": [
+    "{{domain-name}}"
```

Comment:
I believe we don't need this domain name as we just have a Bot with Adaptive Card. Could you please check and remove placeholder?

Reply:
checked it and removed domain part.